app/containers/Forms/Form.tsx

```diff
@@ -19,6 +20,7 @@ import ProductionFields from './ProductionFields';
 interface Props {
   query: Form_query;
   onComplete?: (e: IChangeEvent) => void;
+  onBack?: () => void;
```

Comment:
This prop (and `onComplete` should probably be required)

Reply:
Updated to required